ArmaForces.Boderator/ArmaForces.Boderator.BotService/Controllers/DLCController.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using System.IO;
+using ArmaForces.Boderator.BotService.DataClasses;
+
+namespace ArmaForces.Boderator.BotService.Controllers
+{
+    [Route("users/")]
+    [ApiController]
+    [Produces("application/json")]
+    public class DLCController : ControllerBase
```

Comment:
Is it DLCController with `users` path? You probably should have created 2 controllers. One for users, one for dlc.